### DISCUSSION TITLE
push events: access to default value of a workflow_dispatch inputs event

### DISCUSSION THREAD

**Discussion by hocinemala** [community/community#40435] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

I have a workflow_dispatch with defaults inputs params. The github.event.inputs.<parama> is working when the workflow is manually triggered.
But the github.event.inputs.<parama> is not working on push event.

How to access the default value of inputs parameters when push event?

`on:
  workflow_dispatch:
    inputs:
      command:
        description: 'Select the command'
        required: true
        default: 'change' 
        type: choice
        options:
        - info
        - change
        - fly`

Regards

**Comment by airtower-luna** [comment]
The inputs aren't available. You best option is to provide a default at the right place inside the workflow ("if the input is empty, use this value").

**Reply by hocinemala** [reply]
Ok. Is there an incoming feature to permit to the push event to access inputs?

**Reply by airtower-luna** [reply]
I don't know what GitHub is working on internally, but I'd be extremely surprised if they made something like that. After all, the `push` event has nothing to do with the inputs.

**Comment by redbayme** [comment]
![Screenshot_20221130-202904_Samsung Internet.jpg](https://user-images.githubusercontent.com/118783787/204879908-f2a2a1ae-c487-47de-89fd-cd2889200871.jpg)

**Comment by redbayme** [comment]
- [ ]
